coding/file_container.cpp

```diff
@@ -169,7 +170,7 @@ MappedFile::Handle MappedFile::Map(uint64_t offset, uint64_t size, string const
 #else
   void * pMap = mmap(0, length, PROT_READ, MAP_SHARED, m_fd, alignedOffset);
   if (pMap == MAP_FAILED)
-    MYTHROW(Reader::OpenException, ("Can't map section:", tag, "with [offset, size]:", offset, size));
+    MYTHROW(Reader::OpenException, ("Can't map section:", tag, "with [offset, size]:", offset, size, "errno:", strerror(errno)));
```

Comment:
wait... i agree that text is dependent of platform/library, but errno value... what about macro like EIO, ENODIR (http://www.cplusplus.com/reference/system_error/errc/). I guess their values were standardized.